### DISCUSSION TITLE
upgrade to z-stack or not?

### DISCUSSION THREAD

**Discussion by hs6666** [Koenkk/Z-Stack-firmware#525] [Q&A]
Hi, my current homeassistant attached dongle-p shows following details...

“Generic Zigbee Coordinator (EZSP) Firmware: 6.10.3.0 build 297”

Is there any advantage of installing this instead please , specifically .... 

https://github.com/Koenkk/Z-Stack-firmware/blob/master/coordinator/Z-Stack_3.x.0/bin/CC1352P2_CC2652P_launchpad_coordinator_20240710.zip

?

thanks

**Comment by bobo-cher** [comment]
You have ZBDongle-E with a different chip. Z-stack is for ZBDongle-P.
